webpack/InsightsCloudSync/Components/InsightsTable/InsightsTableActions.js

```diff
@@ -42,6 +44,7 @@ export const fetchInsights = (queryParams = {}) => (dispatch, getState) => {
     search: query,
     sort_by: sortBy,
     sort_order: sortOrder,
+    isSelectAll,
```

Comment:
can you use the snake_case convention we have for the other params?

```suggestion
    select_all: isSelectAll,
```